src/utils/daos_hdlr.c

```diff
@@ -1645,6 +1646,72 @@ fs_copy_file(struct cmd_args_s *ap,
 	return rc;
 }
 
+#define SUB_STR_LEN 256
+static int
+fs_copy_symlink(struct cmd_args_s *ap,
+		struct file_dfs *src_file_dfs,
+		struct file_dfs *dst_file_dfs,
+		struct stat *src_stat,
+		const char *src_path,
+		const char *dst_path)
+{
+	int		rc;
+	daos_size_t	len = 0;
+	char		symlink_value[PATH_MAX-SUB_STR_LEN];
+
+	len = PATH_MAX - SUB_STR_LEN - 1;
```

Comment:
Using "PATH_MAX" leads to compiler warnings about possible string truncation with some compilers, then CI could fail to build since warnings are treated as errors.